akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -175,6 +188,20 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
 
   import ReplicatedDataSerializer._
 
+  private val actorRefFormat: ActorRefFormat = {
+    val conf = system.settings.config
+    val confKey = "akka.cluster.distributed-data.actor-ref-format"
+    // this config is not in reference.conf because it's only temporary for the 2.5.14 release
```

Comment:
My thinking is that the rolling upgrade should just work without any config adjustments, but added this as a backdoor if someone complains about performance regression. In 2.5.14 it will serialize ActorRef to both old and new fields.

Reply:
and we must include in release notes of 2.5.15 that it's not supported to skip 2.5.14 unless full cluster restart is performed

Reply:
makes sense